### PR TITLE
Css frameworks

### PR DESCRIPTION
**Pull request by Sovak3441**
Given the time restriction i have experienced on this task, it is very design limited, simplistic and straight forward. I will continue working on this throughout this weekend, but i'd like some input on the work so far.

### CODE REVIEW

File: src/html/assets/css/custom_styles.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.card {
+  flex-direction: row;
```
Comment (sulenchy) [inline src/html/assets/css/custom_styles.css:2]: There are classes in Bootstrap for flex-direction. See this doc. https://getbootstrap.com/docs/5.3/utilities/flex/#direction

File: src/html/assets/css/custom_styles.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+.card img {
+  width: 30%;
+}
+
+.post-bg {
+  background-color: rgba(11, 11, 11, 0.1)
+}
+
+@media only screen and (max-width: 768px) {
```
Comment (sulenchy) [inline src/html/assets/css/custom_styles.css:12]: You should not be having media queries at all in your custom style. Use Bootstrap breakpoint instead. See https://getbootstrap.com/docs/5.3/layout/breakpoints/#available-breakpoints

File: src/html/index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+  </nav>
+</header>
+<main class="container">
+  <div class="my-5 d-flex justify-content-center">
+    <h1>Please log in to continue.</h1>
+  </div>
+  <div class="row">
+    <div class="errorWrapper"></div>
+    <form>
```
Comment (sulenchy) [inline src/html/index.html:42]: According to the brief of the CA, you are required to have an action attribute with value `./profile`

File: src/scss/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+$blue:    #0d6efd !default;
```
Comment (sulenchy) [inline src/scss/_variables.scss:1]: You don't need the `!default` keyword here. The ! default flag means that a second declaration of this variable will not override the first one.

**Review by sulenchy — CHANGES_REQUESTED** [review]
@Sovak3441 Please, have a look at the comments above. To pass the CA, you need to get them in.

File: src/html/assets/css/custom_styles.css (hunk 2 above)
Comment (Sovak3441) [reply src/html/assets/css/custom_styles.css]: Custom media query removed. Replaced with boostrap breakpoint classes

File: src/html/index.html (hunk 3 above)
Comment (Sovak3441) [reply src/html/index.html]: Added action attribute to form

File: src/html/assets/css/custom_styles.css (hunk 1 above)
Comment (Sovak3441) [reply src/html/assets/css/custom_styles.css]: Removed custom css classes. Replaced with bootstrap classes

File: src/scss/_variables.scss (hunk 4 above)
Comment (Sovak3441) [reply src/scss/_variables.scss]: Removed !default flag in custom _variables.scss file

**Comment by Sovak3441** [comment]
Suggested changes have been implemented.

**Review by sulenchy — APPROVED** [review]

**Pull request merged**